lib/std/private/nimbleutils.nim

```diff
@@ -1,32 +0,0 @@
-##[
```

Comment:
why move this; this is intended to be reusable; moving this to testament/categories makes this un-resuable

Reply:
This contains only one useful proc and that is `actionRetry`, which nothing but testament uses and isn't hard to implement yourself. I really don't think stuff like this belongs into the stdlib, but regardless this is a private module and can be safely removed.

Reply:
I intend to grow this, see https://github.com/dom96/choosenim/issues/192#issuecomment-633935395 (and https://github.com/nim-lang/nimble/issues/803)

the 1st step was to add `nimble dump --json $pkgname` which I've done in https://github.com/nim-lang/nimble/pull/810 (allowing querying nimble package info without a dependency on nimble package); the next step is exposing what's needed for https://github.com/dom96/choosenim/issues/192#issuecomment-633935395

It makes sense to group nimble-related functionality into 1 dedicated module.

Reply:
> Other use cases include extracting the non-nim dependencies from nimble packages and using OS specific package managers to install them.

Nice! I think this is better done as an external "libnimble" though.